vcluster/configure/vcluster-yaml/control-plane/other/advanced/virtual-scheduler.mdx

```diff
@@ -1,41 +1,70 @@
 ---
-title: Virtual Scheduler
+title: Virtual scheduler
 sidebar_label: virtualScheduler
 sidebar_position: 3
 description: Configure virtual scheduling.
 ---
 
 import VirtualSchedulerConfig from '../../../../../_partials/config/controlPlane/advanced/virtualScheduler.mdx'
 
-By default, vCluster reuses the host cluster scheduler to schedule workloads. This saves computing resources, but also has some limitations:
+<!-- vale off -->
 
-1. Labeling nodes inside the virtual cluster has no effect on scheduling, unless [enabling real node syncing](../../../sync/from-host/nodes) with `syncBackChanges` enabled.
-2. Draining or tainting nodes inside the virtual cluster has no effect on scheduling.
+# Virtual scheduler
+
+By default, vCluster reuses the host cluster scheduler to schedule workloads. This conserves compute resources, but also has some limitations:
+
+1. Labeling nodes inside the virtual cluster has **no effect** on scheduling unless [node syncing from the host](../../../sync/from-host/nodes) is enabled using `sync.fromHost.nodes.enabled: true`.
+2. Draining or tainting nodes inside the virtual cluster does **not** affect scheduling.
```

Comment:
I can drain the node when `sync.fromHost.nodes.enabled` is not specified in the vcluster config. But I can't cordon or taint nodes.